### PR TITLE
Feature/namespace module

### PR DESCRIPTION
**Pull request by jaketf**
#11

### CODE REVIEW

**Comment by jaketf** [comment]
@umairidris replied to feedback, if we're all set please tag morgante in LGTM.

File: modules/cdap_namespace/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# CDAP Namespace with Preferences
+
+This module can be used to manage CDAP namespaces with terraform.
+It supports defining a namespace and all it's preferences in a single module call
+## Usage
+
+Basic usage of this module is as follows:
+
+```hcl
+TODO(jaketf)
```
Comment (umairidris) [inline modules/cdap_namespace/README.md]: Fill this in?

File: examples/cdap_namespace/main.tf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+module "staging" {
+  source = "../../modules/cdap_namespace"
+
+  name = var.name
+  preferences = {
+    FOO = "BAR"
+  }
+
```
Comment (umairidris) [inline examples/cdap_namespace/main.tf]: rm extra line

**Comment by umairidris** [comment]
LGTM @morgante good to merge after couple of minor nits

**Comment by jaketf** [comment]
@morgante nits addressed, this should be good to merge now.

**Comment by umairidris** [comment]
With the powers bestowed upon me I can merge this in for you now :D

**Pull request merged**